src/views/PatronPreRegistrationRecordsDuplicates/PatronPreRegistrationRecordsDuplicates.js

```diff
@@ -19,7 +19,7 @@ import { getPatronDuplicatesQuery } from '../../utils';
 import PreRegistrationRecordsDuplicatesList from './PreRegistrationRecordsDuplicatesList';
 
 const PatronPreRegistrationRecordsDuplicates = ({
-  isLoading,
+  isLoading: isLoadingProp,
```

Comment:
isLoadingProp is strange as well :) just call isLoading<What> User? StageUsers? etc

Reply:
try to find :) knowing what it does you can give a better name
BTW if isLoading is used to only disable smth, it means that prop shouldn't sound isLoading, it should be smth like smthDisabled/whatever else